test_e2e/src/couchbase.sh

```diff
@@ -51,6 +51,7 @@ python "$SCRIPT_DIR"/env_setup/couchbase/source_connector/ingest_source_setup_cl
   --bucket_name "$CB_BUCKET" \
   --scope_name "$CB_SCOPE" \
   --collection_name "$CB_COLLECTION" \
+  --id_name "$CB_ID_NAME" \
```

Comment:
don't bother setting this as an env variable. We'd have to add it to the CI secrets. Just set it here as actual text

Reply:
Ah. just saw that is in the constants file. Disregard the above.